locales/FACTER.pot

```diff
@@ -303,18 +303,6 @@ msgstr ""
 msgid "querylv returned success but we got a null LV. WTF?"
 msgstr ""
 
-#. warning
-#: lib/src/facts/aix/kernel_resolver.cc
-msgid "oslevel failed: {1}: kernel facts are unavailable"
-msgstr ""
-
-#. warning
-#: lib/src/facts/aix/kernel_resolver.cc
-msgid ""
-"Could not parse rml cache even after regenerating with oslevel: kernel facts "
-"are unavailable. Try running 'oslevel -s' to debug."
-msgstr ""
-
```

Comment:
Looks like it failed to get update last time (notice 3.6.4 project version). Updating this file is not automated yet for C++ projects, so it only happens if a developer with gettext tooling on their machine builds the code before submitting the PR. My guess is whoever removed those strings didn't have the tooling installed.